packages/fxa-settings/src/lib/glean/index.ts

```diff
@@ -351,8 +352,30 @@ const recordEventMetric = (
     case 'cad_mobile_pair_use_app_view':
       cadMobilePairUseApp.view.record();
       break;
-    case 'account_pref_recovery_key_submit':
-      accountPref.recoveryKeySubmit.record();
+    case 'account_pref_view':
+      accountPref.view.record();
+      break;
+      case 'account_pref_recovery_key_submit':
+        accountPref.recoveryKeySubmit.record();
+        break;
```

Comment:
I thought something would auto-format this...anyways nit: indentation

Reply:
🤔 would have thought so too. I'll fix before merging